### ISSUE TITLE
Cura 4.0 BETA

### ISSUE THREAD

**Issue by Ondrashek** [nallath/BarbarianPlugin#1]
In Cura 3.4, 3.6, 4.0 if is checked "Scale extremely small models" in Cura settings, Cura scales model to 10000% and your plugin scales to 25400% instead 2540%.

**Comment by nallath** [comment]
Yes. That's because it simply scales the selected models by 25.4 (It does it relatively instead of absolute).

**Comment by Ondrashek** [comment]
> Yes. That's because it simply scales the selected models by 25.4 (It does it relatively instead of absolute).

It would be great, if your script will scale model to 100% and then to 2540%.
